test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -0,0 +1,375 @@
+const { strict: assert } = require('assert');
+
+const {
+  convertToHexValue,
+  withFixtures,
+  connectDappWithExtensionPopup,
+  getWindowHandles,
+} = require('../helpers');
+
+describe('Create token, approve token and approve token without gas', function () {
+  describe('Add a custom token from a dapp', function () {
+    const ganacheOptions = {
+      accounts: [
+        {
+          secretKey:
+            '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+          balance: convertToHexValue(25000000000000000000),
+        },
+      ],
+    };
+
+    it('creates, imports and renders the balance for the new token', async function () {
+      await withFixtures(
+        {
+          dapp: true,
+          fixtures: 'imported-account',
```

Comment:
Can we use the `connected-state` fixture, instead of using `imported-account` fixture and the `connectDappWithExtensionPopup` method?